ui/components/app/signature-request/signature-request.container.js

```diff
@@ -16,18 +21,31 @@ function mapStateToProps(state, ownProps) {
   const hardwareWalletRequiresConnection =
     doesAddressRequireLedgerHidConnection(state, from);
   const isLedgerWallet = isAddressLedger(state, from);
+  const currentNetwork =
```

Comment:
See my comment on the other container file regarding network name

Reply:
done